core/src/main/java/com/sequenceiq/cloudbreak/service/image/ImageService.java

```diff
@@ -208,7 +155,8 @@ private StatedImage checkIfBasePermitted(StatedImage image, boolean baseImageEna
         return image;
     }
 
-    private Set<String> getSupportedOperationSystems(Long workspaceId, ImageSettingsV4Request imageSettings, String clusterVersion, String platform) throws Exception {
+    private Set<String> getSupportedOperationSystems(Long workspaceId, ImageSettingsV4Request imageSettings, String clusterVersion, String platform)
```

Comment:
Can you please rename it to `getSupportedOperatingSystems`?

Reply:
sure